packages/build-utils/src/monorepos/monorepo-managers.ts

```diff
@@ -0,0 +1,37 @@
+import type { Framework } from '@vercel/frameworks';
+
+/**
+ * The supported list of monorepo managers.
+ *
+ * This list is designed to work with the @see {@link detectFramework} function.
+ *
+ * @example
+ *   import { monorepoManagers as frameworkList } from '@vercel/build-utils'
+ *   import { detectFramework } from '@vercel/build-utils'
+ *
+ *   const fs = new GitDetectorFilesystem(...)
+ *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
+ *
+ * @todo Will be used by the detect-eligible-projects API endpoint for a given git url.
```

Comment:
Will this `@todo` be removed after https://github.com/vercel/api/issues/12179 is done? If so, can we add a note on that issue to make sure this is removed later?

(I'm still learning how different teams are tracking tasks. Thanks!)

Reply:
oh shoot - ill just remove this! good catch